packages/appstore/src/assets/svgs/banner/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
and shall we rename this file to` index.ts`, which simply re-exports the `AppstoreBannerImage` component?
we'll just follow the current standards? 🚒